install oxpy into the oxDNA installation directory instead of the user or Python installation directory
author: Miguel Dias Costa (National University of Singapore)
--- oxDNA-3.5.2/oxpy/CMakeLists.txt.orig	2023-06-28 15:22:31.870578035 +0800
+++ oxDNA-3.5.2/oxpy/CMakeLists.txt	2023-06-28 18:05:27.313737196 +0800
@@ -62,7 +62,7 @@
 	if(OxpySystemInstall)
 		INSTALL(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip install ${OXPY_OUTPUT_DIR})")
 	else()
-		INSTALL(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip install --user ${OXPY_OUTPUT_DIR})")
+		INSTALL(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pip install --prefix=${CMAKE_INSTALL_PREFIX} ${OXPY_OUTPUT_DIR})")
 	endif()
 	
 ENDIF(Python)
